### PR TITLE
Feature/add UUID

### PR DESCRIPTION
**Pull request by nickmccally**
If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What's this PR do?

Check if uuids are missing from user documents. Add uuid for users, where missing, from auth service. If one does not exist on auth, one will be generated by orange-api. 
 
#### Related JIRA tickets:

- https://jira.amida.com/browse/ORANGE-481

#### How should this be manually tested?

Run `yarn && yarn start`. You should run this *AFTER* you have already started the auth service on the branch that has uuid support. 

#### Any background context you want to provide?

Support new microservices using uuid versus username (which is typically registration email)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,11 +57,15 @@
     "passport-jwt": "3.0.1",
     "pg": "6.1.6",
     "pg-hstore": "2.3.2",
+    "request": "false2.88.0",
```
Comment (ElijahGeorge) [inline package.json]: @nickmccally I think we can remove the `false`. It seems to be a yarn bug

File: package.json [hunk 2]
```diff
@@ -55,11 +57,15 @@
...
+    "request-promise": "false4.2.2",
```
Comment (ElijahGeorge) [inline package.json]: dito

**Review by ElijahGeorge — COMMENTED** [review]
Added some comments pertaining to minor changes. LGTM

**Review by ElijahGeorge — CHANGES_REQUESTED** [review]
Everything looks good to me except the `false` in `package.json`.

**Review by ElijahGeorge — APPROVED** [review]

**Review by mountHouli — APPROVED** [review]
It worked for me. I dropped my messaging DB and created a new one from scratch, then tested sharing two users logs and sent a message.

**Pull request merged**